addon/components/sortable-group.js

```diff
@@ -88,18 +115,28 @@ export default Component.extend({
   */
   update() {
     let sortedItems = this.get('sortedItems');
-    let y = this._itemPosition;
+    let position = this._itemPosition;
 
     // Just in case we haven’t called prepare first.
-    if (y === undefined) {
-      y = this.get('itemPosition');
+    if (position === undefined) {
+      position = this.get('itemPosition');
     }
 
     sortedItems.forEach(item => {
+      let dimension;
+
       if (!get(item, 'isDragging')) {
-        set(item, 'y', y);
+        set(item, this.get('direction'), position);
+      }
+
+      if (this.get('direction') === 'x') {
+       dimension = 'width';
       }
-      y += get(item, 'height');
+      if (this.get('direction') === 'y') {
+       dimension = 'height';
+      }
+
+      position += get(item, dimension);
```

Comment:
Seeing as `this.get('direction')` is used 3 times in the method body, could we assign it to a local let up top?

```js
let direction = this.get('direction')
```